src/main/scala/org/apache/spark/sql/delta/metering/DeltaLogging.scala

```diff
@@ -18,7 +18,6 @@ package org.apache.spark.sql.delta.metering
 
 import scala.util.Try
 import scala.util.control.NonFatal
-
```

Comment:
can you remove this spurious change please

Reply:
done